.github/workflows/on_push.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: PyNE
+
+on:
+  push:
+    branches: [ "master", "main" ]
```

Comment:
There is no "main" branch in this repository. I'm surprised this doesn't throw an error. 
I'm perfectly happy for "master" to be renamed to "main" as is the norm now. However, there is no reason to have both.